### PR TITLE
TiDev Banner Copy Updates

### PR DESCRIPTION
**Pull request by joshualambert**
This PR updates the Titanium banner that displays to:

1. Display TiDev as the project copyright maintainer.
2. Promote the Titanium Slack community.
3. Promote the TiDev Liberapay link.

It displays like this on iTerm+ZSH like this:

Screen Shot 2022-03-23 at 6 50 31 PM

### CODE REVIEW

**Comment by caspahouzer** [comment]
I’m missing more donation possibilities. Please let us setup GitHub sponsors asap

File: lib/logger.js [hunk 1]
```diff
@@ -96,7 +96,10 @@ logger.banner = function () {
 	var info = appc.pkginfo.package(module, 'version', 'about');
 	if (bannerEnabled) {
 		logger.log(info.about.name.cyan.bold + ', CLI version ' + info.version + (logger.activeSdk ? ', Titanium SDK version ' + logger.activeSdk : '') + '\n' + info.about.copyright);
-		logger.log('\n' + __('Please report bugs to %s', 'https://github.com/appcelerator/titanium_mobile/issues'.cyan) + '\n');
+		logger.log('\n' + __('- Please report bugs here: %s', 'https://github.com/tidev/titanium_mobile/issues'.cyan));
+		logger.log(__('- Remember! Titanium is a community supported open-source project and relies on donations for on-going development.'));
```
Comment (m1ga) [inline lib/logger.js:100]: I would remove the `Remember!` from the line

File: package.json [hunk 2]
```diff
@@ -2,11 +2,11 @@
 	"about": {
 		"name": "Titanium Command-Line Interface",
 		"author": "Appcelerator",
-		"copyright": "Copyright (c) 2012-2020, Appcelerator, Inc.  All Rights Reserved.",
+		"copyright": "Copyright (c) 2012-2022, TiDev, Inc. All Rights Reserved.",
 		"id": "com.appcelerator.titanium.cli"
 	},
 	"name": "titanium",
-	"description": "Appcelerator Titanium Command line",
+	"description": "TiDev Titanium Command line",
```
Comment (cb1kenobi) [inline package.json:9]: No need to say TiDev here.

File: package.json [hunk 3]
```diff
@@ -2,11 +2,11 @@
 	"about": {
 		"name": "Titanium Command-Line Interface",
 		"author": "Appcelerator",
-		"copyright": "Copyright (c) 2012-2020, Appcelerator, Inc.  All Rights Reserved.",
+		"copyright": "Copyright (c) 2012-2022, TiDev, Inc. All Rights Reserved.",
```
Comment (cb1kenobi) [inline package.json:5]: I have been waiting to discuss the copyright before changing it. I'm not a lawyer, so I don't know if we can change it, or if we can, maybe we just reset the year to 2022?

File: lib/logger.js [hunk 4]
```diff
@@ -96,7 +96,10 @@ logger.banner = function () {
...
+		logger.log(__('- Support the project here: %s', 'https://liberapay.com/tidev'.cyan));
```
Comment (cb1kenobi) [inline lib/logger.js:101]: I profoundly disagree with putting any mention of donations in the CLI output. I've already said time and time again, Titanium cannot sustain off donations. Titanium is open source. It is free. The only way Titanium will survive is if we get people to contribute. You set a precedence when you ask for donations to fund development. It's unfair to all the contributors that work on Titanium for free. Soon, nobody will want to contribute for free.

File: lib/logger.js [hunk 5]
```diff
@@ -96,7 +96,10 @@ logger.banner = function () {
...
+		logger.log(__('- Join the TiDev Slack community here: %s \n', 'https://tislack.org'.cyan));
```
Comment (cb1kenobi) [inline lib/logger.js:102]: Why are we putting the Slack URL in the CLI output? This banner output is already too big, too many lines. I would suggest just displaying a link to https://titaniumsdk.com. That's where the docs are. That's where we want to funnel people to go first for help.

File: package.json [hunk 6]
```diff
@@ -2,11 +2,11 @@
...
 		"id": "com.appcelerator.titanium.cli"
 	},
 	"name": "titanium",
-	"description": "Appcelerator Titanium Command line",
+	"description": "TiDev Titanium Command line",
 	"keywords": [
 		"appcelerator",
```
Comment (cb1kenobi) [inline package.json:11]: I've been removing "appcelerator" from the keywords.

**Review by cb1kenobi — CHANGES_REQUESTED** [review]

**Review by caspahouzer — APPROVED** [review]
We have to MAKE MONEY. This is one way to generate attention

**Comment by caspahouzer** [comment]
We should change the donation link to a page on tidev.io or titaniumsdk.com as soon as we have multiple donation possibilities

**Comment by m1ga** [comment]
> We have to MAKE MONEY. This is one way to generate attention

I would just keep it at one line with a link to titaniumsdk.com (or github). In my opinion this is not the place to ask for donations. 
If you build an app it will scroll through anyways and you don't see it. If I want to see the help for a command I don't want to be bothered with that lines all the time.

Add it as a post-install message, so every new installation will be presented with a message, but not in every command.

**Comment by cb1kenobi** [comment]
I agree with @m1ga. As stated before, I would not like to see a donation link in the CLI. However, if we add a donation link to the CLI, we should point it to a distinct CLI-specific URL that we can tie into analytics to see if anyone actually goes to the link.

Also, you can no longer have post install scripts print messages after installing an npm package. npm 7 and newer silence stdout and there's no way I have come up with around it. We'd have to do a first run thing.

**Comment by caspahouzer** [comment]
Are there any news for this? @joshualambert

**Pull request closed without merging**